anthos-bm-gcp-terraform/scripts/setup.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+DATE=$(date)
+ZONE_FLAG="zone"
```

Comment:
This script is GCE startup specific, and doesn't conform to the uniform Terraform way. Recommend we look into Terraform null_resource [1]. This approach will also allow us to consider making changes to google provider if that makes sense. 

[1] https://registry.terraform.io/providers/hashicorp/null/latest/docs/resources/resource

Reply:
We have removed dependency on startup script from this commit onwards: https://github.com/GoogleCloudPlatform/anthos-samples/pull/11/commits/c88fb01463469fde55b56925cd04158c40dcc02a